### PR TITLE
MM-401 Diagrams Don't Work for Non-Square Images

### PR DESCRIPTION
**Pull request by HussainHaider**

### CODE REVIEW

File: src/Canvas/index.tsx [hunk 1]
```diff
@@ -56,6 +56,7 @@ export interface CanvasProps {
   canvasColor: string;
   backgroundImage: string;
   exportWithBackgroundImage: boolean;
+  preserveBackgroundImageAspectRatio: string;
```
Comment (shemigon) [inline src/Canvas/index.tsx:59]: What is the point of setting this property from the outside? Is going to change?

**Review by prcolin — COMMENTED** [review]
When I tried to build this in `findings` I got the following error:

```
ERROR in C:\Projects\findings\src\Images\FreeHandEditDiagramCanvas\index.tsx
./src/Images/FreeHandEditDiagramCanvas/index.tsx 126:15-32
[tsl] ERROR in C:\Projects\findings\src\Images\FreeHandEditDiagramCanvas\index.tsx(126,16)
      TS2741: Property 'preserveBackgroundImageAspectRatio' is missing in type '{ className: string; ref: MutableRefObject<ReactSketchCanvasRef>; strokeWidth: number; strokeColor: string; backgroundImage: string; exportWithBackgroundImage: true; withTimestamp: true; keepScale: true; onChange: () => void; }' but required in type 'ReactSketchCanvasProps'.
ts-loader-default_e3b0c44298fc1c14
 @ ./src/Images/Images.tsx 31:36-74
 @ ./src/AppTabs.tsx 36:17-43
 @ ./src/app.tsx 7:18-36
 @ ./src/index.tsx 8:14-28

webpack 5.75.0 compiled with 1 error in 2610 ms
```

Am I missing something here?

File: src/Canvas/index.tsx (hunk 1 above)
Comment (HussainHaider) [reply src/Canvas/index.tsx:59]: I did it so that we can change the value in the future if needed

**Comment by HussainHaider** [comment]
> When I tried to build this in `findings` I got the following error:
> 
> ```
> ERROR in C:\Projects\findings\src\Images\FreeHandEditDiagramCanvas\index.tsx
> ./src/Images/FreeHandEditDiagramCanvas/index.tsx 126:15-32
> [tsl] ERROR in C:\Projects\findings\src\Images\FreeHandEditDiagramCanvas\index.tsx(126,16)
>       TS2741: Property 'preserveBackgroundImageAspectRatio' is missing in type '{ className: string; ref: MutableRefObject<ReactSketchCanvasRef>; strokeWidth: number; strokeColor: string; backgroundImage: string; exportWithBackgroundImage: true; withTimestamp: true; keepScale: true; onChange: () => void; }' but required in type 'ReactSketchCanvasProps'.
> ts-loader-default_e3b0c44298fc1c14
>  @ ./src/Images/Images.tsx 31:36-74
>  @ ./src/AppTabs.tsx 36:17-43
>  @ ./src/app.tsx 7:18-36
>  @ ./src/index.tsx 8:14-28
> 
> webpack 5.75.0 compiled with 1 error in 2610 ms
> ```
> 
> Am I missing something here?

I guess I need to add this prop in the finding repo. let me check it

**Comment by HussainHaider** [comment]
> 

I have created a new PR https://github.com/SARC-MedIQ/findings/pull/52. Kindly this PR with it

**Review by prcolin — APPROVED** [review]
@shemigon Please review in conjunction with https://github.com/SARC-MedIQ/findings/pull/52

**Pull request closed without merging**